zk.go

```diff
@@ -5,13 +5,17 @@ import (
 	"path"
 	"sort"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/samuel/go-zookeeper/zk"
 )
 
 // ZK wraps a zookeeper connection
-type ZK struct{ *zk.Conn }
+type ZK struct {
+	*zk.Conn
+	chrootPath string
```

Comment:
Oh, right, it's OK then, just sounds a bit strange in my opinion. Maybe that's just me :)